File: chat-sdk/src/main/java/ru/usedesk/chat_sdk/internal/data/framework/httpapi/IHttpApiLoader.java
package ru.usedesk.chat_sdk.internal.data.framework.httpapi;

import androidx.annotation.NonNull;

import java.io.IOException;

import ru.usedesk.chat_sdk.external.entity.UsedeskOfflineForm;

public interface IHttpApiLoader {
    void post(@NonNull String baseUrl, @NonNull UsedeskOfflineForm offlineForm) throws IOException;
}

File: chat-sdk/src/main/java/ru/usedesk/chat_sdk/internal/data/framework/httpapi/IHttpApiLoader.kt
package ru.usedesk.chat_sdk.internal.data.framework.httpapi

import ru.usedesk.chat_sdk.external.entity.old.UsedeskOfflineForm
import java.io.IOException

interface IHttpApiLoader {
    @Throws(IOException::class)
    fun post(baseUrl: String, offlineForm: UsedeskOfflineForm)
}